### DISCUSSION TITLE
app should link to this group

### DISCUSSION THREAD

**Discussion by jidanni** [community/community#2814] [Mobile] [answered]
The app hasn't added this feedback group yet. It only has a link to send an email.
![Screenshot_20201128-141239](https://user-images.githubusercontent.com/1230959/100495217-261fe280-3184-11eb-930e-8035428156ee.jpg)

**Comment by candyho** [comment, marked as answer]
Hey thanks for your feedback. The Discussions feature isn't natively supported on mobile hence we haven't redirected feedback from in-app to here yet. Good news is that we're actively working on Discussions support on mobile so please stay tuned!
